### PR TITLE
Update Muon MVA ID model and working points in MiniAOD and NanoAOD

### PR DESCRIPTION
**Pull request by andrea21z**
#### PR description:
- This PR includes the update of the Muon MVA ID working points in MiniAOD and NanoAOD for the latest training. 

- This training is the final and has been published in CMS-PAS-MUO-22-001.

- Related PR: to include the new MVA model in data repository, in .onnx format -> https://github.com/cms-data/RecoMuon-MuonIdentification/pull/9

#### PR validation:
- We check the output of the new branch and Working Points running runTheMatrix and everything looks fine.
- We execute the basic tests  suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by clacaputo** [comment]
type muon

**Comment by clacaputo** [comment]
enable nano

**Comment by andrea21z** [comment]
Hi @clacaputo, it seems to have failed due to an 'input DAS' error, should I do something else?

**Comment by clacaputo** [comment]
test parameters:

- enable = nano
- pull_request = https://github.com/cms-data/RecoMuon-MuonIdentification/pull/9

**Comment by clacaputo** [comment]
please test

**Comment by clacaputo** [comment]
+reconstruction

- update Muon MVA ID working points
- new model available in https://github.com/cms-data/RecoMuon-MuonIdentification/pull/9
- reco differences as expected

File: PhysicsTools/NanoAOD/python/muons_cff.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     inputTensorName= cms.string("float_input"),
     outputNames = cms.vstring(["probGOOD", "wpMedium", "wpTight"]),
     batch_eval =cms.bool(True),
-    outputFormulas = cms.vstring(["at(1)", "? at(1) > 0.14 ? 1 : 0", "? at(1) > 0.35 ? 1 : 0"]),
+    outputFormulas = cms.vstring(["at(1)", "? at(1) > 0.08 ? 1 : 0", "? at(1) > 0.20 ? 1 : 0"]),
```
Comment (simonepigazzini) [inline PhysicsTools/NanoAOD/python/muons_cff.py:43]: Should we preserve the old formula for Run2 reprocessing by adding it back through the `run2_nanoAOD_ANY` modifier?

Comment (simonepigazzini) [reply PhysicsTools/NanoAOD/python/muons_cff.py:43]: ok cool

**Comment by simonepigazzini** [comment]
we have a backlog of few PR while busy with the xpog workshop. We will review and sign next monday.

File: PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc [hunk 2]
```diff
@@ -991,18 +991,16 @@ void PATMuonProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
     float mvaID = 0.0;
     constexpr int MVAsentinelValue = -99;
     constexpr float mvaIDmediumCut = 0.08;
-    constexpr float mvaIDtightCut = 0.12;
+    constexpr float mvaIDtightCut = 0.20;
```
Comment (vlimant) [inline PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc:994]: could we open a PR with pushing these cuts to configuration level ?

Comment (vlimant) [reply PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc:994]: you may indeed.

**Comment by clacaputo** [comment]
please test

**Comment by clacaputo** [comment]
+reconstruction

- https://github.com/cms-sw/cmssw/pull/41903#issuecomment-1600580145

**Comment by vlimant** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**